book.rb

```diff
@@ -0,0 +1,16 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    Rental.new(date, person, self)
+  end
```

Comment:
- [ ] Great work implementing the changes suggested by the previous reviewer. There is a little change you need to make. Due to the parameters passed to the `initialize` method of the `Rental` class, there would be an error when creating a new instance of the `Rental` class in the `add_rental` method of the `Book` class. In the `initialize` method of the `Rental` class, you passed `date`, `book` and `person`: ` def initialize(date, book, person)` therefore when creating the instance in the Book class, it should be `Rental.new(date, self, person)`. Kindly correct this. 👍 😎

Reply:
Thank you, I missed this key.